stardis/io/model/util.py

```diff
@@ -0,0 +1,54 @@
+from pathlib import Path
+import pandas as pd
+
+
+PATH_TO_ASPLUND_2009 = Path(__file__).parent / "data" / "asplund_2009_processed.csv"
+ASPLUND_DEFAULT_HELIUM_MASS_FRACTION_Y = (
+    0.2492280  # The Asplund 2009 mass fraction of measured He
+)
+ASPLUND_DEFAULT_HEAVY_ELEMENTS_MASS_FRACTION_Z = (
+    0.01337  # The Asplund 2009 mass fraction of measured heavy metals
+)
+
+
+def create_scaled_solar_profile(
+    atom_data,
+    Y=ASPLUND_DEFAULT_HELIUM_MASS_FRACTION_Y,
```

Comment:
i meant rename the variables helium_mass_fraction_y

Reply:
It is done